modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/MultiSurfaceTypeBinding.java

```diff
@@ -20,6 +20,7 @@
 import java.util.List;
 import javax.xml.namespace.QName;
 import org.geotools.gml3.GML;
+import org.geotools.gml3.MultiSurface;
```

Comment:
Uh, this is a problem, think it's going to prevent parsing/handling of curved elements in curved surfaces.
See ``org.geotools.geometry.jts.MultiSurface``

Reply:
OK, I will try with **org.geotools.geometry.jts.MultiSurface**. Can I modify this class to override the method getGeometryType ?
The geometry is parsed as a Multipolygon and The FESParseEncodeUtil uses the geometryType to define the name.

Reply:
You can try to change that, make sure it's not going to break existing tests.